### PR TITLE
Update free-courses-en.md

### PR DESCRIPTION
**Pull request by chessboards**
Adds a NodeJS section and online course.

## What does this PR do?
Add Resource(s)

## For resources
free-courses-en.md
### Description 
Creates a new section for node.js along with a course.

### Why is this valuable (or not)
It is highly valuable, as there was not previously a course listing about this topic. The course added is also very high quality.

### How do we know it's really free?
It is hosted on Youtube.

### For book lists, is it a book?
It is a course.

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by chessboards** [comment]
check_urls=free-courses-en.md

File: free-courses-en.md [hunk 1]
```diff
@@ -278,7 +284,7 @@
 
 ### Sails.Js
 
-* [Develop web apps in Node.js and Sails.js](https://courses.platzi.com/courses/develop-apps-sails-js/)
+* [Develop web apps in free-programming-playgrounds.md.js and Sails.js](https://courses.platzi.com/courses/develop-apps-sails-js/)
```
Comment (eshellman) [inline free-courses-en.md:287]: ???

Comment (eshellman) [reply free-courses-en.md:287]: maybe move to Javascript

File: free-courses-en.md [hunk 2]
```diff
@@ -265,6 +266,11 @@
 * [Bento jQuery Track](https://bento.io/topic/jquery) (Bento)
 
 
+#### NodeJS
+
+* [Node.js Basics](https://www.youtube.com/watch?v=hQd4TCGB_nE&list=PLqq-6Pq4lTTa-d0iZg41U2RDqECol9C5B) - Java Brains (Youtube)
```
Comment (eshellman) [inline free-courses-en.md:271]: we prefer playlist links for youtube

File: free-courses-en.md (hunk 1 above)
Comment (chessboards) [reply free-courses-en.md:287]: not sure what that is, I don't think I touched that

**Pull request merged**